java/Alena-Vorokh/src/com/zoolatech/lecture1/tasks/_9/Task_9.java

```diff
@@ -0,0 +1,24 @@
+package com.zoolatech.lecture1.tasks._9;
+
+/* Write a program that accepts an integer value and outputs all its digits in the reverse order on a new line.*/
+
+import java.util.Scanner;
+
+public class Task_9 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input value: ");
+        int N = scanner.nextInt();
+        int i = 1;
+        int j;
+        int a = 10;
+        int length = String.valueOf(N).length();
+        while (i <= length ) {
+            j = (N % a);
+            N = (N - j)/a;
```

Comment:
you don't need to subtract j from N here, the division operation will remove any remainders:
```
15 % 10 = 5
15 / 10 = 1 (same as (15 - 5) / 10)
```